### PR TITLE
Cleanup spandx config to make it easier to connect to a local api

### PR DESCRIPTION
**Pull request by dkuc**
This change makes it easy to switch what API you are talking to.  Just change the `apiHost` in the spandx.config.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,11 @@ The follow six branches are used by IF
 A push or merge to master will automatically release to ci-beta and qa-beta
 The same will happen with action against master-stable, it will automatically release to ci-stable and qa-stable
 The prod-beta and prod-stable environments are updated by a deliberate push, (to each branch)
+
+### Nuggets and Tidbits
+##### Start insights-proxy directly with docker
+``` shell
+ docker run -v $PWD/config:/config --rm --net='host' -p1337:1337 -e PLATFORM=linux -ti docker.io/redhatinsights/insights-proxy
```
Comment (AllenBW) [inline README.md]: Is this only for use with a local api?

Comment (dkuc) [reply README.md]: No. That's for anyone who does want to clone/install insights-proxy.

Comment (AllenBW) [reply README.md]: Not seeing success on a mac, might want to specify the platform

Comment (dkuc) [reply README.md]: Good call.  Done.

**Pull request merged**